src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = 40;
+  const numberOfDays = days;
+  const DISCOUNT = 20;
```

Comment:
The name 'DISCOUNT' could be more descriptive. It should reflect that it's a discount for a short term rental, such as 'SHORT_TERM_DISCOUNT'.